var/spack/repos/builtin/packages/py-heat/package.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyHeat(PythonPackage):
+    """Heat is a flexible and seamless open-source software for high performance data analytics
+    and machine learning. It provides highly optimized algorithms and data structures for tensor
+    computations using CPUs, GPUs and distributed cluster systems on top of MPI."""
+
```

Comment:
Can you re-add the homepage again?

Reply:
Of course, I can! Should it be the GitHub or PyPI page?